Fleece/PlatformCompat.hh

```diff
@@ -35,13 +35,19 @@
     // WARNING: sizeof() will not work on this array since it's actually declared as a pointer.
     #define StackArray(NAME, TYPE, SIZE)    TYPE* NAME = (TYPE*)_malloca(sizeof(TYPE)*(SIZE))
 
-    #define MKDIR(PATH, MODE) ::_mkdir(PATH)
-    #define chmod  ::_chmod
     #define fdopen ::_fdopen
     #define fseeko fseek
     #define ftello ftell
-	#define strncasecmp _strnicmp
-	#define strcasecmp _stricmp
+    #define strncasecmp _strnicmp
+    #define strcasecmp _stricmp
+
+   int mkdir_u8(const char* const path, int mode);
```

Comment:
If you declare real functions here, there should be a `namespace fleece` wrapped around the contents of this header, so they don't end up in the global namespace.